apps/cms.c

```diff
@@ -908,13 +912,15 @@ int cms_main(int argc, char **argv)
 
             settable = EVP_PKEY_CTX_settable_params(pctx);
```

Comment:
An example is in evp_enc.c.. 
       /* Check the cipher actually understands this parameter */
        if (OSSL_PARAM_locate_const(EVP_CIPHER_settable_ctx_params(c->cipher),
                                    OSSL_CIPHER_PARAM_KEYLEN) == NULL)
            return 0;

Reply:
There is also a OSSL_PARAM_locate

Reply:
Applied. Many thanks for the clue!